fastlane/Fastfile

```diff
@@ -447,28 +447,53 @@ platform :ios do
   lane :generate_docs do
     ENV["INCLUDE_DOCC_PLUGIN"] = "true"
     version_number = current_version_number
-    docs_folder = "docs/#{version_number}"
-    hosting_base_path = "purchases-ios/#{version_number}"
-
-    Dir.chdir("..") do
-      sh("swift",
-         "package",
-         "--disable-sandbox",
-         "--allow-writing-to-directory",
-         docs_folder,
-         "generate-documentation",
-         "--target",
-         "RevenueCat",
-         "--disable-indexing",
-         "--output-path",
-         docs_folder,
-         "--hosting-base-path",
-         "#{hosting_base_path}",
-         "--transform-for-static-hosting")
+    hosting_base_path = "purchases-ios-docs/#{version_number}"
+    docs_repo_base_url = "https://github.com/revenuecat/"
```

Comment:
Should this be `ssh` instead (`"git@github.com:RevenueCat/" + docs_repo_name`)

Reply:
that's a great idea! I'll try it out in a follow-up PR

Reply:
Can we make this an environment variable and the default it to either `https://github.com/revenuecat/` or `git@github.com:RevenueCat/` if not set?